src/utils.ts

```diff
@@ -0,0 +1,20 @@
+import { randomUUID } from "crypto"
+
+const usedIds = new Set<string>()
+export const getFreeId = () => {
```

Comment:
please specify the return type

Reply:
See https://github.com/paritytech/opstooling-js/pull/4#discussion_r873445107 on the question about the linting rule.